src/domain/audio/AudioInput.ts

```diff
@@ -258,10 +258,33 @@ class AudioInput {
             sampleRate: AudioConstants.SAMPLE_RATE
         });
 
-        this.#_audioStreamSource = this.#_audioContext.createMediaStreamSource(this.#_audioInput);
+        try {
+            // Firefox throws an exception here
+            // Uncaught (in promise) DOMException: AudioContext.createMediaStreamSource:
+            // Connecting AudioNodes from AudioContexts with different sample-rate is currently not supported.
+            //
+            // Which is likely a bug and not an actual missing implementation.
+            this.#_audioStreamSource = this.#_audioContext.createMediaStreamSource(this.#_audioInput);
+        } catch {
+            // We work around by using a context without sample rate requirement and handling re-sampling in the input worklet.
+            this.#_audioContext = new AudioContext();
```

Comment:
```suggestion
            // We work around by using a context without sample rate requirement and handling re-sampling in the input worklet.
            console.log("Use resampler to match sample rates.");
            this.#_audioContext = new AudioContext();
```
Log a comment to indicate that the warning logged is being addressed.